src/main/java/com/zufar/onlinestore/product/entity/ProductInfo.java

```diff
@@ -52,30 +52,18 @@ public boolean equals(Object object) {
         if (object == null || getClass() != object.getClass())
             return false;
         ProductInfo that = (ProductInfo) object;
-        return active == that.active &&
-                Objects.equals(id, that.id) &&
-                Objects.equals(name, that.name) &&
-                Objects.equals(description, that.description) &&
-                Objects.equals(price, that.price) &&
-                Objects.equals(currency, that.currency) &&
-                Objects.equals(quantity, that.quantity);
+        return Objects.equals(id, that.id);
```

Comment:
What about class org.apache.commons.lang3.builder.ToStringBuilder, it runs faster because under the hood there are many improvements to increase performance